### PR TITLE
Feat: external link icon

### LINKED ISSUE #1471: External Link Icon - Add external link icon to Footer link(s)

### LINKED ISSUE #1478: External Link Icon - Eligibility Index - External link icon to VA.gov

### LINKED ISSUE #1484: External link icon - Help page

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #1471 

Also closes #1478 , closes #1484 

This PR assumes that all external links have an attribute of `target="_blank"` and adds an icon after the link.

### Footer links

image

### Eligibility index
image

### Help page
image

### CODE REVIEW

**Comment by angela-tran** [comment]
We'll need a separate ticket to update the Help copy

File: benefits/static/css/styles.css [hunk 1]
```diff
@@ -142,6 +142,25 @@ a:visited:not(.btn) {
   color: var(--selected-color);
 }
 
+[target="_blank"]::after {
```
Comment (thekaveman) [inline benefits/static/css/styles.css:145]: I think at a minimum we want to add the element here:

```suggestion
a[target="_blank"]::after {
```

Is targeting all links with `_blank` enough? Too much? Is it consistent practice to make external links open in new tabs?

Asking Sarah some related questions on the Figma board:

* https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3?node-id=9234:110446&mode=design#500359107
* https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3?node-id=9234:110446&mode=design#500359437

**Review by thekaveman — COMMENTED** [review]
This is looking great! Small comment below and some questions out to Design, but this seems like a clean way to do it.

**Review by thekaveman — APPROVED** [review]
👍

**Pull request merged**